maestro-ios/src/main/java/ios/idb/IdbIOSDevice.kt

```diff
@@ -394,6 +396,7 @@ class IdbIOSDevice(
                         if (launchArguments.isNotEmpty()) {
                             appArgs.addAll(launchArguments)
                         }
+                        sessionId = maestroSessionId?.toString() ?: UUID.randomUUID().toString()
```

Comment:
IDB does not support sessionId. idb.proto is not for IDB, which makes everything confusing. We'll improve that soon. For now lets not pass it here, because IDB will break if we do (this code is not unused anyway).